.travis.yaml

```diff
@@ -8,3 +8,4 @@ script:
   - tox
 notifications:
 email: rpappalax@gmail.com 
+
```

Comment:
Eventually a Dockerfile will be needed here, but it's OK to remove this one if you wish.  Actually, I have one from Benny and the sync tests that will allow desktop selenium testing that may be a better starting place for this.

Reply:
I'll revisit the whole Docker issue later on when I have a better plan in place for what I want to do